examples/integrations/microstream/pom.xml

```diff
@@ -0,0 +1,38 @@
+<!--
+
+    Copyright (c) 2021 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.helidon.examples.integrations</groupId>
+        <artifactId>helidon-examples-integrations-project</artifactId>
+        <version>2.4.0-SNAPSHOT</version>
+    </parent>
+
+    <groupId>io.helidon.examples.integrations.microstream</groupId>
+    <artifactId>helidon-examples-integrations-microstream-project</artifactId>
+    <name>Helidon Microstream Integration Examples</name>
+    <packaging>pom</packaging>
+
+    <modules>
+        <module>greetings-se</module>
+        <module>greetings-mp</module>
+    </modules>
+</project>
```

Comment:
I noticed some files have no new line at the end of the file.

Reply:
unless this is discovered by checkstyle, we do not need to fix this. Maybe this would call for a change in checkstyle configuration